hedera-node/src/main/resources/bootstrap.properties

```diff
@@ -52,23 +52,27 @@ balances.nodeBalanceWarningThreshold=0
 cache.records.ttl=180
 contracts.allowAutoAssociations=false
 contracts.allowCreate2=true
+contracts.chainId=295
 contracts.defaultLifetime=7890000
+contracts.freeStorageTierLimit=100
```

Comment:
Isn't this 100M ?

Reply:
Tried to describe in [new javadoc for `ContractStoragePriceTiers`](https://github.com/hashgraph/hedera-services/blob/03300-decouple-contract-storage-fees/hedera-node/src/main/java/com/hedera/services/fees/charging/ContractStoragePriceTiers.java#L66); contracts with fewer than `contracts.freeStorageTierLimit` key/value pairs are not charged any rent.